test/e2e/data/infrastructure-vsphere-govmomi/main/vgpu/kustomization.yaml

```diff
@@ -0,0 +1,6 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
no need for this as long as there are no e2e tests, and we don't have gpu's in our e2e test-environment.